hutch_python/tests/test_tstpython.py

```diff
@@ -3,8 +3,11 @@
 from subprocess import STDOUT, CalledProcessError, check_output
 
 import pytest
+from packaging import version
```

Comment:
`packaging` isn't part of the standard library (https://pypi.org/project/packaging/) and must be a dependency-of-a-dependency if it's not in our requirements files.

Given that it's just in the test suite, we should probably add it to dev-requirements

Reply:
I went around looking for a standard library solution to this, but it seems that the similar `distutils.version.LooseVersion` is being deprecated in favor of `packaging.version`.  I'll add this to dev-requirements